v2/azure-arm.yaml

```diff
@@ -870,16 +870,11 @@ status:
 #         $generatedConfigs:
 #           BlobEndpoint: $.Status.PrimaryEndpoints.Blob
 #
-# $export: <bool>
-#     Requests that support for this resource type be generated.
-#     Automatically includes all other types required for this resource
-#     If not specified, the default value is false.
-#     Only valid for resource types.
 #
 # $exportAs: <string>
 #     Requests that this resource type be exported with the specified
 #     identifier, allowing poorly named types to be renamed.
-#     Implies $export: true
+#     Automatically exports the resource type.
```

Comment:
I think you can just delete this line - the comment above already states "Requests that this resource type be exported", we don't need to repeat that.